### ISSUE TITLE
UCF101

### ISSUE THREAD

**Issue by GuoQuanhao** [whwu95/MVFNet#3]
Hi, Can you provide UCF101 configs?

**Comment by whwu95** [comment]
To finetune the UCF101 with Kinetics400 pretrained model, our configs are similar to the Kinetics400. The only difference is the learning rate and learning schedule as follows.

# optimizer
optimizer = dict(type='SGD', lr=0.001, momentum=0.9,
                 weight_decay=0.0005, nesterov=True)
optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))
# learning policy
lr_config = dict(
    policy='step',
    step=[10, 20])
total_epochs = 25
